game-core/src/main/java/games/strategy/engine/chat/ChatMessagePanel.java

```diff
@@ -68,8 +68,16 @@
               chat.updateStatus(status.trim());
             }
           });
+  private final ChatSoundProfile chatSoundProfile;
 
-  public ChatMessagePanel(final Chat chat) {
+  /** A profile defines the sounds to use for various chat events. */
+  public enum ChatSoundProfile {
+    LOBBY,
+    GAME
+  }
+
+  public ChatMessagePanel(final Chat chat, final ChatSoundProfile chatSoundProfile) {
```

Comment:
Any thoughts/consideration to using a factory pattern to hide the `ChatSoundProfile` enum?
eg:
```
public static ChatMessagePanel newLobbyChatPanel() { .. }
public static ChatMessagePanel newGameChatPanel() { .. }
```

Reply:
This wouldn't be easy to achieve: The `ChatSoundProfile` is passed a couple of layers down:
https://github.com/triplea-game/triplea/pull/5318/files#diff-9aeb0682c4d70742aca95a41ebca9d8bL27-R30

So a factory pattern would actually make things more complicated.

Reply:
Okay. The 'passed' a couple layers down is a problem in of itself, too many layers sit on top and certainly was just 'wired' through. Perhaps this is a place where DI would help.

#Nothing further needed.